### PR TITLE
docs(react-scheduler): add the scheduler Appointment Tooltip guide

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

**Comment by ushkal** [comment]
I think we should split this guide in 2 - "Appointment Form" and "Appointment Tooltip" (or just "Tooltip"). It's not obvious from the title what this guide is about. Separate guides will be more easy to find. Plus the customization part is too abstract now

**Comment by MaximKudriavtsev** [comment]
@ushkal, yeah you are right. But now we have big plans to rewrite AppointmentForm plugin and I don't want to describe it's behavior now. I suppose just rename this guide to `Appointment Tooltip` and remove information about `AppointmentForm` plugin, and describe it in the future.

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### Uncontrolled Mode
+
+The following demo demonstrates simple way to display an appointment data.
+
+.embedded-demo({ "path": "scheduler-tooltip/uncontrolled", "showThemeSelector": true })
+
+### Controlled Mode
+
+In controlled mode, you should control the all/particular `AppointmentTooltip's` state. Pass the `visible` property and `onVisibilityChange`  event to control the visibility externally. Pass the [appointmentMeta](../reference/appointment-tooltip.md#appointmentmeta) property and `onAppointmentMetaChange` event to control the display data externally.
```
Comment (ushkal) [inline packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: What does all/particular mean?

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+.embedded-demo({ "path": "scheduler-tooltip/controlled", "showThemeSelector": true })
+
+### Customization
+
+To override default markup or add your own, you should define a `xxxComponent` properties, all list of these properties is available at plugin [API reference page](../reference/appointment-tooltip.md). In the following demo shows how to use `AppointmentTooltip's` `headerComponent` and `contentComponent` properties to make custom markup.
```
Comment (ushkal) [inline packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: all list of these properties > a list of these properties

Comment (ushkal) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: In the following demo shows -> The following demo shows

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# React Scheduler - Appointment Tooltip
+
+The React Scheduler component's `AppointmentTooltip` plugin displays compact appointment data.
+
+## Related Plugins
+
+The following plugins display an appointment data:
```
Comment (ushkal) [inline packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: Appointment data -> appointment tooltip ?

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md (hunk 1 above)
Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: The component has 2 state properties, you can change them both and one by one.

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# React Scheduler - Appointment Tooltip
+
+The React Scheduler component's `AppointmentTooltip` plugin displays compact appointment data.
```
Comment (ushkal) [inline packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: I think it's better to replace 'compact' with 'detailed' and to mention that an appointment tooltip can contain some action buttons

Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: `Detailed` means `additional information`. `Compact` means `little information`. I think we should keep this word.

Comment (ushkal) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: An appointment tooltip as a concept is often a panel with summarized info and actions - a date/time, description, resource name, and so on. From the other hand, appointments are often kept as minimalistic as possible to not distract an attention, e.g. in month view we render only a caption not date/time. So we can say that a tooltip displays extended/additional information :)

Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: Okay, you are right)

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# React Scheduler - Appointment Tooltip
+
+An appointment tooltip displays brief information about the appointment and contains buttons that manage the appointment.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]:
```suggestion
An appointment tooltip can display information about the appointment and contains buttons that manage the appointment.
```

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## Related Plugins
+
+The following plugins are required:
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: Why 'plugins'? I can only see one here.

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+- [AppointmentTooltip](../reference/appointment-tooltip.md) - displays the appointment tooltip
+
+## Basic Usage
+
+Import the plugin listed above. A user will be able to invoke the appointment tooltip by a click on an appointment.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]:
```suggestion
Import the plugin listed above. A user can click on an appointment to show its tooltip.
```

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+### Uncontrolled Mode
+
+The following demo shows the use of the appointment tooltip in uncontrolled mode:
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: Why 'the use of the'?

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+.embedded-demo({ "path": "scheduler-tooltip/uncontrolled", "showThemeSelector": true })
+
+### Controlled Mode
+
+In controlled mode, the state of all or particular `AppointmentTooltip`s should be controlled externally. To control their visibility, pass the `visible` property and `onVisibilityChange` event to the plugin. To control the displayed data, pass the [appointmentMeta](../reference/appointment-tooltip.md#appointmentmeta) property and `onAppointmentMetaChange` event.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: 'all or particular'?

Comment (dirk-pieterse) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]:
```suggestion
In controlled mode, the state of `AppointmentTooltip`s should be controlled externally. To control their visibility, pass the `visible` property and `onVisibilityChange` event to the plugin. To manage the displayed data, pass the [appointmentMeta](../reference/appointment-tooltip.md#appointmentmeta) property and `onAppointmentMetaChange` event.
```

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+.embedded-demo({ "path": "scheduler-tooltip/controlled", "showThemeSelector": true })
+
+### Customize the Appearance
+
+To customize the appointment tooltip's appearance, redefine `xxxComponent` properties. `xxx` stands for the name of the element that you want to customize. For example, the tooltip's header and content are customized using the `headerComponent` and `contentComponent` properties, as shown in the code below. Refer to the [AppointmentTooltip API reference page](../reference/appointment-tooltip.md) for a list of all `xxxComponent` properties.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]:
```suggestion
To customize the tooltip's appearance, specify the `xxxComponent` properties (`xxx` is the name of the element you want to customize). For example, the `headerComponent` and `contentComponent` properties are used to customize the tooltip's header and content, as shown in the code below. Refer to the [AppointmentTooltip API reference page](../reference/appointment-tooltip.md) for a list of all `xxxComponent` properties.
```

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md (hunk 5 above)
Comment (RomanTsukanov) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]:
```suggestion
An appointment tooltip displays information about the appointment and contains buttons that manage the appointment.
```

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md (hunk 6 above)
Comment (RomanTsukanov) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]:
```suggestion
The following plugin is required:
```

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md (hunk 8 above)
Comment (RomanTsukanov) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]:
```suggestion
The following demo shows how to use the appointment tooltip in uncontrolled mode:
```

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md (hunk 7 above)
Comment (RomanTsukanov) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: Ok

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md (hunk 9 above)
Comment (RomanTsukanov) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: Ok

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md (hunk 10 above)
Comment (RomanTsukanov) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: Ok

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md (hunk 5 above)
Comment (dirk-pieterse) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: Ok

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md (hunk 6 above)
Comment (dirk-pieterse) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: Ok

File: packages/dx-react-scheduler/docs/guides/appointment-tooltip.md (hunk 8 above)
Comment (dirk-pieterse) [reply packages/dx-react-scheduler/docs/guides/appointment-tooltip.md]: Ok

**Pull request merged**